.travis.yml

```diff
@@ -45,7 +45,8 @@ before_install:
           -Dgcs-resource-test-bucket=gcp-storage-resource-bucket-sample
           -Dgcs-read-bucket=gcp-storage-bucket-sample-input
           -Dgcs-write-bucket=gcp-storage-bucket-sample-output
-          -Dgcs-local-directory=/tmp/gcp_integration_tests/integration_storage_sample";
+          -Dgcs-local-directory=/tmp/gcp_integration_tests/integration_storage_sample
+          -Dspring.cloud.stream.function.definition=none";
```

Comment:
Can we just do it on the sample that is broken rather than for the whole project build on Travis?
Or I'm missing something?

Reply:
Sure; I'll add the property directly to sample.